src/main/java/vg/civcraft/mc/citadel/command/Acid.java

```diff
@@ -105,4 +110,17 @@ public boolean execute(CommandSender sender, String[] args) {
 	public List<String> tabComplete(CommandSender sender, String[] args) {
 		return new ArrayList<>();
 	}
+
+	public boolean isChestBeingAcidBlocked (Block block) {
```

Comment:
A method name like this makes it sound like this just checks something, but it actually actively drops stuff. Definitely should be changed.

Reply:
canDropChestContents more suited?

Reply:
No, that also sounds like a getter. Just make all of it a breakBlock() method and include the handling for non-inventories